### PR TITLE
[API-698] Clean up activities endpoints and documentation

### PR DESCRIPTION
**Pull request by zackisaacs**
This is the last one.... Note that the detailed activity representations are identical for a couple of the endpoints -- I just copied them, so no need to re-review the detailed activity responses.

Adds sample responses and cleans up the documentation for each of the activities endpoints. I'm primarily looking for english typos and data sanitization in review.

Completes [API-698](https://strava.atlassian.net/browse/API-698)

### CODE REVIEW

File: swagger/swagger.json.mustache [hunk 1]
```diff
@@ -751,6 +817,211 @@
...
+                      "hazardous": false,
+                      "starred": false
+                    },
+                    "kom_rank": null,
+                    "pr_rank": null,
+                    "achievements": [],
+                    "hidden": false
+                  }
+                ],
+                "splits_metric": [
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: We show splits on a ride????

File: swagger/swagger.json.mustache [hunk 2]
```diff
@@ -831,79 +1307,217 @@
         ],
         "responses": {
           "200": {
-            "description": "Activities of the athlete",
+            "description": "The authenticated athlete's activities",
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: period missing

File: swagger/swagger.json.mustache [hunk 3]
```diff
@@ -1020,6 +1666,38 @@
...
+                  "id": 12345678987654321,
+                  "activity_id": 12345678987654321,
+                  "post_id": null,
+                  "resource_state": 2,
+                  "text": "Good job and keep the cat pictures coming!",
+                  "mentions_metadata": null,
+                  "created_at": "2018-02-08T19:25:39Z",
+                  "athlete": {
+                    "id": 186707,
+                    "username": "psagan",
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: Yikes, we return too much info here, gonna add that to `opd2`

File: swagger/swagger.json.mustache [hunk 4]
```diff
@@ -158,6 +158,72 @@
...
+                      "min": 350,
+                      "time": 80
+                    },
+                    {
+                      "max": 450,
+                      "min": 400,
+                      "time": 81
+                    },
+                    {
+                      "max": -1,
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: max above is 0, just making sure these are consistent with what we return

File: swagger/swagger.json.mustache [hunk 5]
```diff
@@ -790,6 +1061,211 @@
...
+                    "distance": 9434.8,
+                    "start_index": 211,
+                    "end_index": 2246,
+                    "average_cadence": 78.6,
+                    "device_watts": true,
+                    "average_watts": 237.6,
+                    "segment": {
+                      "id": 673683,
+                      "resource_state": 2,
+                      "name": "Broadway/51st up Tunnel to Skyline/Grizzly",
```
Comment (MateoAO) [inline swagger/swagger.json.mustache]: Intresting segment name to pick. Why not Tunnel Rd?!?!?!

File: swagger/swagger.json.mustache (hunk 1 above)
Comment (zackisaacs) [reply swagger/swagger.json.mustache]: Apparently.

File: swagger/swagger.json.mustache (hunk 2 above)
Comment (zackisaacs) [reply swagger/swagger.json.mustache]: Thanks.

File: swagger/swagger.json.mustache (hunk 3 above)
Comment (zackisaacs) [reply swagger/swagger.json.mustache]: :bta:

**Pull request merged**